### PR TITLE
Audio progress bar and Play by select

### PR DESCRIPTION
**Pull request by Vansh-Baghel**
[Play Music by select #1](https://github.com/debanjana-a11y/music_player/issues/1) ✅
[Add audio progress bar #3](https://github.com/debanjana-a11y/music_player/issues/3) ✅
* Changed the CSS of the song list on hover.
* Added audio progress bar.
* Made the song play on select.
* Can select the progress bar and change the song progress accordingly.
* Kept the play button as play while changing the song.

### CODE REVIEW

File: script.js [hunk 1]
```diff
@@ -1,79 +1,92 @@
...
+  currentSongName = currentSongUl.innerText;
+  music = new Audio(`./audio/${currentSongName}.mp3`);
+
+  currentSongUl.classList.add("playing");
+}
+
+function selectMusic() {
+  stopMusic();
+
+  let li = event.target.closest("li");
```
Comment (debanjana-a11y) [inline script.js:83]: "event" is deprecated, we should avoid it. right ?
Also we can pass this event by "onclick="selectMusic(this)"" from html and use it in js.

File: script.js [hunk 2]
```diff
@@ -1,79 +1,92 @@
...
+  let nodes = Array.from(songList.children);
```
Comment (debanjana-a11y) [inline script.js:84]: This line #84 is not needed, we can remove this.

**Review by debanjana-a11y — CHANGES_REQUESTED** [review]

File: script.js (hunk 1 above)
Comment (Vansh-Baghel) [reply script.js:83]: It won't be an issue as it's working fine. 
Still, if you don't want it, then just convert event -> this.event in JS. 
There's no need to add  selectMusic(**this**)

File: script.js (hunk 2 above)
Comment (Vansh-Baghel) [reply script.js:84]: Yes, those were 2 ways of doing it which I wanted to show . I just forgot to comment.

**Comment by Vansh-Baghel** [comment]
A suggestion!! Do add hover cursor pointer on progressBar.

**Pull request merged**